<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailedwindow</class>
 <widget class="QWidget" name="detailedwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>31</y>
     <width>194</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Todo</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>5</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Tomalue</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>45</y>
     <width>441</width>
     <height>481</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>23</green>
         <blue>23</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>15</green>
         <blue>30</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>23</green>
         <blue>23</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>15</green>
         <blue>30</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>15</green>
         <blue>30</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>45</y>
     <width>441</width>
     <height>481</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>23</green>
         <blue>23</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>15</green>
         <blue>30</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>23</red>
         <green>23</green>
         <blue>23</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>15</green>
         <blue>30</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>196</red>
         <green>15</green>
         <blue>30</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::NoPen</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
